tests/Fixture/FixtureAttributeCommand.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/*
+ * This file is part of the zenstruck/console-test package.
+ *
+ * (c) Kevin Bond <kevinbond@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Zenstruck\Console\Test\Tests\Fixture;
+
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\ConsoleOutputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * @author Kevin Bond <kevinbond@gmail.com>
+ */
+#[AsCommand(
+    name: 'fixture:command',
+    description: 'zenstruck/console-test command for tests'
+)]
+final class FixtureAttributeCommand extends Command
```

Comment:
I have copy past your other FixtureCommand but without cleaning it  
Maybe test should be duplicate for this case (with #[AsCommand] attribute) ?